crates/aranya-client-capi/ctest/test_team.c

```
@@ -0,0 +1,1722 @@
+    
+    /* Initialize logging subsystem */
+    AranyaError err = aranya_init_logging();
+    if (err != ARANYA_ERROR_SUCCESS) {
+        fprintf(stderr, "Failed to initialize logging: %s\n", aranya_error_to_str(err));
+        return EXIT_FAILURE;
+    }
+    
+    printf("Running aranya-client-capi team tests\n");
+    printf("=====================================\n");
+
+    /* Spawn 3 daemons for multi-user testing if path provided */
+    if (argc == 2) {
+        const char *daemon_path = argv[1];
+        
+        /* Kill any stray daemon processes from previous runs */
+        printf("Cleaning up any stray daemon processes...\n");
+        system("pkill -TERM -f 'aranya-daemon.*test-team' 2>/dev/null || true");
+        sleep_ms(500);
+        system("pkill -KILL -f 'aranya-daemon.*test-team' 2>/dev/null || true");
+        
+        /* Spawn owner daemon */
+        printf("Spawning owner daemon: %s\n", daemon_path);
+        owner_pid = spawn_daemon_at(daemon_path, "/tmp/team-run-owner", "test-team-owner", "/team-owner", 41001);
+        printf("Owner Daemon PID: %d\n", owner_pid);
+        
+        /* Spawn member daemon */
+        printf("Spawning member daemon: %s\n", daemon_path);
+        member_pid = spawn_daemon_at(daemon_path, "/tmp/team-run-member", "test-team-member", "/team-member", 41002);
+        printf("Member Daemon PID: %d\n", member_pid);
+        
+        /* Spawn device daemon */
+        printf("Spawning device daemon: %s\n", daemon_path);
+        device_pid = spawn_daemon_at(daemon_path, "/tmp/team-run-device", "test-team-device", "/team-device", 41003);
+        printf("Device Daemon PID: %d\n", device_pid);
+        
+        /* Wait for all daemons to initialize */
+        printf("Waiting 15 seconds for daemons to initialize...\n");
+        sleep_ms(15000);
+        printf("Daemons should be ready now\n");
+    }
+
+    /* Test basic structures (don't need daemon) */
+    if (test_team_id() != 0) exit_code = 1;
```

This seems like it is attempting to run all the tests as if it is a test suite. That's what `ctest` is for. Individual tests should return an error right away if there is any failure.